service/routers/decks.js

```diff
@@ -19,7 +20,7 @@ const getDecks = async (req, res) => {
     res.sendStatus(500)
   }
 }
-
+// allowed to  all
```

Comment:
Is it though? From the assignment:
_A user should only have access to create, update and delete their **own** cards and decks._